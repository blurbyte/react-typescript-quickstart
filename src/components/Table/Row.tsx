import styled from 'styled-components';

const Row = styled.tr`
  background: rgba(40, 40, 40, 0.02);
  transition: all 0.2s ease;

  &:hover {
    background: rgba(74, 108, 187, 0.2);
  }
`;

export default Row;
