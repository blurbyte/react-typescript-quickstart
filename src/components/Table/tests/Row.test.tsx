import * as React from 'react';
import { shallow } from 'enzyme';

import Row from '../Row';

describe('Row', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Row />);
    expect(wrapper).toMatchSnapshot();
  });
});
