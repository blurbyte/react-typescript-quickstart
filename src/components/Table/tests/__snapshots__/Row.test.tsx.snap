// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders correctly 1`] = `
.c0 {
  background: rgba(40,40,40,0.02);
  -webkit-transition: all 0.2s ease;
  transition: all 0.2s ease;
}

.c0:hover {
  background: rgba(74,108,187,0.2);
}

<tr
  className="c0"
/>
`;
